docs/maintain-guides-how-to-chill.md

```diff
@@ -0,0 +1,32 @@
+---
+id: maintain-guides-how-to-chill
+title: Chilling
+sidebar_label: Chilling
+---
+
+Stakers can assume any one of the three states: validating, nominating, or chilling. When a staker
+does not want to actively engage in staking - but does not want to unbond their funds either - they
+can choose to temporarily become inactive while keeping their funds staked. The act of stopping
+either nominating or validating is known as _chilling_.
+
+An account can step back from participating in active staking by calling the `chill` extrinsic in
+the [staking pallet][chill extrinsic]. When an account chooses to chill, they will become inactive
+in the next era. The call must be signed by the _controller_ account, not the _stash_.
+
+## Chilling as a Nominator
+
+When you chill after being a nominator, your nominations will be reset. This means that when you
+decide to start nominating again you will need to select validators to nominate once again. These
```

Comment:
To go from chilling to being active again, is there any steps that should be taken? Or can I just start nominating or validating again?

Reply:
It might help @lsaether to include these details in this section here, but what do you think?

Reply:
Yeah, might help to explicitly mention it.